aws/resource_aws_dx_private_virtual_interface.go

```diff
@@ -84,6 +84,16 @@ func resourceAwsDxPrivateVirtualInterface() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
+			"mtu": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"jumbo_frame_enabled": {
```

Comment:
Nitpick - In the [documentation](https://docs.aws.amazon.com/directconnect/latest/UserGuide/set-jumbo-frames-vif.html) the feature is called _Jumbo Frame**s**_. For consistency I suggest naming the attribute `jumbo_frames_enabled` or `jumbo_frame_capable` (to match the API).

Reply:
Oops, I meant to have this match the API but I just spaced on it.  This will be corrected in my next commit.